js/renovation/ui/data_grid/data_grid.tsx

```diff
@@ -16,22 +27,24 @@ export const viewFunction = ({ widgetRef, props: { className }, restAttributes }
 
 @Component({
   defaultOptionRules: null,
-  // jQuery: { register: true },
+  jQuery: { register: true },
   view: viewFunction,
 })
 export class DataGrid extends JSXComponent(DataGridProps) {
   @Ref()
   widgetRef!: HTMLDivElement;
 
+  @InternalState() widget: any;
```

Comment:
May be getter instead of state (it avoid unnecessary render on startup)
private get widget() {
  return LegacyDataGrid.getInstance(this.widgetRef);
}